frontend/javascripts/oxalis/constants.js

```diff
@@ -45,13 +44,14 @@ export type OrthoViewRects = $ReadOnly<OrthoViewMap<Rect>>;
 
 export const ArbitraryViewport = "arbitraryViewport";
 export const ArbitraryViews = {
-  arbitraryViewport: "arbitraryViewport",
-  TDView: "TDView",
+  arbitraryViewport: { id: "arbitraryViewport", name: "Arbitrary View", short: "Arbitrary" },
+  TDView: { id: "TDView", name: "3D", shortName: "3D" },
 };
```

Comment:
I'm not sure whether this is the best way to do this. The downside I see is that this stores objects (which are constants) within the store, even though this isn't really necessary. Additionally, it complicates all usages of the OrthoView variable.
I assume you wanted a way to access the name/short name for an ortho view? I think I would have gone for something like:

```
const OrthoViewToName = {
  PLANE_XY: "XY",
  ...
}
```

That way, the store can still contain the primitive ids.

Reply:
Ok, I just scrolled through the diff. It would also be quite easier to review this PR if there weren't that many `PLANE_XY.id` changes :see_no_evil: Do you see a strong advantage in your approach over my suggestion? If not: could you please follow my suggestion? Then, the amount of line changes would be greatly reduced probably.

Reply:
>Do you see a strong advantage in your approach over my suggestion?

I do not see a strong advantage of this approach. My thought was just that this way we could have the internally used id and the name that is exposed to the user at the same place. The `shortName` was a little too much as there is currently no need for it. My idea behind this was the discussed case on what to do when we have too many items in a row so not all tabs are visible. In that case, we could use short names to make more tabs visible instead of hiding some of them in a menu.

I will go for a `OrthoViewToName` constant object :+1: